README.md

```diff
@@ -0,0 +1,60 @@
+#  NoiteDev
+[![Build Status](https://travis-ci.org/condessalovelace/mavenquickstart.svg?branch=master)](https://travis-ci.org/condessalovelace/mavenquickstart) ![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=br.com%3Amavenquickstart&metric=alert_status)
+
+## Description 
+
+This project illustrates an example of a simple registrer of students for learning.
+
+## Demostration
+
+![prototype](./img/prototype.png)
+
+## Starting
+
+To run the project, you will need to install the following programs:
+
+- [MySQL: a toolkit, necessary to carry out the project] (https://www.mysql.com/)
+- [XAMPP: a server to upload the database] (https://www.apachefriends.org/pt_br/index.html)
+- [Nodejs: Required to create the project] (https://nodejs.org/en/)
+- [VS Code: For project development] (https://code.visualstudio.com/)
+
+## Development
+
+To start the development, it is necessary to clone the GitHub project in a directory of your choice:
+
+```
+cd "Diretory of your preference"
+git clone https://github.com/Suspir0n/noiteDev.git
+```
+
+### Construction
+
+To build the project with Nodejs, execute the commands below:
+
+```
+npm init
+```
+
+The command will download the file called * package.json *.
+
+With that, MySQL would be missing. For such action, execute the command below:
+
+```
+npm install mysql --save
+```
+
+The command will download all the dependencies you need to use this tool.
+
+## Features
+
+The project can be used as a template to start the development of a Javascript project using Nodejs. It also demonstrates in a practical way how to create a Student Register and also demonstrate in a table in nodejs quickly and easily.
+
+## Configuration
+
+To execute the project, it is necessary to use VsCode or an IDE of your preference, so that it identifies the dependencies necessary for execution in the repository. Once the project is imported, it will be possible to test its functionality in real time.
```

Comment:
```suggestion
To execute the project, it is necessary to use VS Code or an IDE of your preference, so that it identifies the dependencies necessary for execution in the repository. Once the project is imported, it will be possible to test its functionality in real time.
```